admin_guide/manage_scc.adoc

```diff
@@ -429,56 +429,44 @@ To provide additional capabilities:
 === Modify Cluster Default Behavior
 
 To modify your cluster so that it does not pre-allocate UIDs, allows containers
-to run as any user, and prevents privileged containers:
+to run as any user, but prevents privileged containers, grant access to the
+*anyuid* SCC for everyone:
 
-[NOTE]
-====
-In order to preserve customized SCCs during upgrades, do not edit settings on
-the default SCCs other than priority, users, groups, labels, and annotations.
-====
-
-. Edit the *restricted* SCC:
-+
 ----
- $ oc edit scc restricted
+ $ oc adm policy add-scc-to-group anyuid system:authenticated
```

Comment:
This is not identical to editing `restricted` for the following reasons:
- `anyuid` has priority and will win other SCCs most of the time
- `anyuid` doesn't drop some UID-related capabilities that `restricted` drops
- `anyuid` also allows any `fsGroups`

I agree that granting `anyuid` to everyone could also lead to unexpected results and even break the cluster components in some cases (see https://github.com/openshift/openshift-ansible/pull/8558). But here, I think that it less painful than silent reset to the defaults during upgrade.

Ideally, I'd rather remove this chapter at all, because granting access for `system:authenticated` to any SCC may be a breaking change and IMHO it's a bad practice. Cluster admins should always create custom SCCs and grant access to them only to users/groups which are really need this access.

I decided to keep this chapter because it's about "how to go fast and ignore security best practices", so it's all big hack for someone who doesn't care/doesn't have time to fully understand/just making it work.

Reply:
add a NOTE section about this?

Reply:
@pweil- I've added a WARNING section instead.

PTAL if you find a minute for me :)